helix-view/src/clipboard.rs

```diff
@@ -96,6 +96,37 @@ pub fn get_clipboard_provider() -> Box<dyn ClipboardProvider> {
     Box::new(provider::FallbackProvider::new())
 }
 
+// Check if we are in tmux and the allow-passthrough is on.
+fn tmux_allow_passthrough() -> bool {
+    use helix_stdx::env::env_var_is_set;
+
+    if !env_var_is_set("TMUX") {
+        return false;
+    }
```

Comment:
Then this check would be redundant

Reply:
We still need this check. The reason is that this function is also used in the OSC52 fallback, which won't necessarily be used with tmux. So, to escape OSC52 string through tmux, we still need to check if we are in tmux and whether the feature is on or not.